src/utils/wallet.ts

```diff
@@ -63,7 +63,7 @@ export class Wallet {
     // generate(account.address, { small: true });
 
     console.log(info('Retriving account information'))
-    console.log('‍📬 Your address ' + account.address)
+    console.log(info('‍📬 Your wallet address '), account.address)
```

Comment:
Can we also return DID here so it'll be easy for the user.

Reply:
Actually, we are returning the DID in the `connectUser` method.